welkin/__version__.py

```diff
@@ -4,7 +4,7 @@
 __title__ = "welkin"
 __description__ = "Python Welkin Health API Wrapper."
 __url__ = "https://github.com/Lightmatter/welkin-health"
-__version__ = "0.0.7"
+__version__ = "0.1.0"
 __author__ = "Sam Morgan"
 __author_email__ = "sam@lightmatter.com"
```

Comment:
Should update this email?

Reply:
Probably a good call. Should this go to a generic `@lightmatter.com` email instead?

All this field does is populate the author `mailto:` link on [pypi](https://pypi.org/project/welkin/):
<img width="282" alt="image" src="https://github.com/Lightmatter/welkin-health/assets/356814/6a50739e-7867-4a08-a79a-e6eeb776162a">

I'm happy to stay as the primary contact and keep authorship, using my personal gmail account here. Otherwise, the Author name should be "Lightmatter" with an email address of your choice.